config/releases.exs

```diff
@@ -110,7 +110,13 @@ config :hackney,
 
 # Configure Lambda-based actions
 
-config :meadow, :lambda, digester: {:lambda, "meadow-digester"}
+config :meadow, :lambda,
+  digester: {:lambda, "meadow-digester"},
+  exif: {:lambda, "meadow-exif"}
 
 config :sequins, Actions.GenerateFileSetDigests,
   queue_config: [processor_concurrency: 50, visibility_timeout: 300]
+
+config :sequins, Actions.ExtractExifMetadata,
+  queue_config: [processor_concurrency: 50, visibility_timeout: 300],
+  notify_on: [CopyFileToPreservation: [status: :ok], ExtractExifMetadata: [status: :retry]]
```

Comment:
@mbklein should I skip the `notify_on` parameter here as in `Actions.GenerateFileSetDigests` above?

Reply:
Yes. The notification config is in `sequins.exs` and will be carried over to here. `releases.exs` only needs to override the `queue_config`.

Reply:
Ok thanks, removed.